src/style.css

```diff
@@ -0,0 +1,38 @@
+* {
```

Comment:
i mess it first time, but you do not need to reset padding, only need to reset margin in body, will be better to add this style to body